Configs/.config/waybar/modules/idle_inhibitor.jsonc

```diff
@@ -1,8 +1,8 @@
     "idle_inhibitor": {
         "format": "{icon}",
         "format-icons": {
-        "activated": "󰥔",
-        "deactivated": ""
+        "activated": "󰥔 ",
+        "deactivated": " "
         }
```

Comment:
wont this increase the space between the icon and the clock? each module is padded already.